src/ctl/src/common/hummock_service.rs

```diff
@@ -80,11 +82,17 @@ impl HummockServiceOpts {
                 bail!(MESSAGE);
             }
         };
+        let use_new_object_prefix_strategy = match env::var("RW_USE_NEW_OBJECT_PREFIX_STRATEGY") {
+            Ok(use_new_object_prefix_strategy) => use_new_object_prefix_strategy == "true",
```

Comment:
I missed this in the previous review. `use_new_object_prefix_strategy` should be treated like `data_dir` because when it is incorrectly set, hummock won't work. In this case, I think we should pass `use_new_object_prefix_strategy` explicitly for `HummockServiceOpts::from_env` just like `data_dir`. This means we need to add `use_new_object_prefix_strategy` (default to true) for all ctl commands that requires `HummockServiceOpts::from_env` (in this PR you only add it for sst dump)

Reply:
That make sense, fixed by passing `use_new_object_prefix_strategy` explicitly for all ctl command.